src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -229,38 +223,38 @@ public synchronized void check() {
         }
     }
 
-    public synchronized void nurDoppelteAnzeigen(boolean index) {
-        // zum Debuggen: URLs die doppelt sind, in die History eintragen
-        // damit sie markiert werden
-        DatenFilm film;
-        HashSet<String> hashDoppelt = new HashSet<>();
-        HashSet<String> hash = new HashSet<>();
-        Iterator<DatenFilm> it = this.iterator();
-        while (it.hasNext()) {
-            film = it.next();
-            if (index) {
-                if (!hash.contains(film.getIndex())) {
-                    hash.add(film.getIndex());
-                } else {
-                    // dann ist er mind. doppelt in der Liste
-                    hashDoppelt.add(film.arr[DatenFilm.FILM_URL]);
-                }
-            } else if (!hash.contains(film.arr[DatenFilm.FILM_URL])) {
-                hash.add(film.arr[DatenFilm.FILM_URL]);
-            } else {
-                // dann ist er mind. doppelt in der Liste
-                hashDoppelt.add(film.arr[DatenFilm.FILM_URL]);
-            }
-        }
-        it = this.iterator();
-        while (it.hasNext()) {
-            if (!hashDoppelt.contains(it.next().arr[DatenFilm.FILM_URL])) {
-                it.remove();
-            }
-        }
-        hash.clear();
-        hashDoppelt.clear();
-    }
+//    public synchronized void nurDoppelteAnzeigen(boolean index) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)